src/main/webapp/cache-status/cache-status.html

```diff
@@ -109,7 +109,33 @@ <h2 class="card-pf-title">
 
     <div class="row row-cards-pf">
       <!-- BEGIN CARD -->
-      <div class="col-xs-2 col-sm-2 col-md-2" match-height="ispn-card">
+      <div class="col-xs-6 col-sm-4 col-md-3" match-height="ispn-card"
+           ng-if="currentCacheStats.firstServerStats['offline-sites'].length > 0
```

Comment:
as sugested by @tristantarrant, I've changed the names of the attributes:
https://github.com/infinispan/infinispan/pull/3948#discussion_r51123203

the current names are ```sites-online```, ```sites-offline``` and ```sites-mixed```